package.json

```diff
@@ -6,7 +6,10 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "prettier:check": "prettier --check .",
+    "prettier:fix": "prettier --write .",
+    "fix": "bun run prettier:fix"
```

Comment:
Is this intentional? I don't see bun being used anywhere else, and it isn't installed in local dev (just CI) 🤔